tools/congestion-model/src/strategy/mod.rs

```diff
@@ -1,15 +1,15 @@
 use crate::model::ChunkExecutionContext;
 use crate::{QueueId, ShardId};
 
-pub use fancy_global_transaction_stop::FancyGlobalTransactionStop;
 pub use global_tx_stop::GlobalTxStopShard;
+pub use nep::NepStrategy;
 pub use new_tx_last::NewTxLast;
 pub use no_queues::NoQueueShard;
 pub use simple_backpressure::SimpleBackpressure;
 pub use traffic_light::TrafficLight;
 
-mod fancy_global_transaction_stop;
 mod global_tx_stop;
+mod nep;
```

Comment:
Are you trying to delete `fancy_global_transaction_stop` or was that a mistake? I think you left the file in but removed the link to it. Probably it should stay in and stay properly linked, right=?